modules/nf-core/gmmdemux/meta.yml

```
@@ -0,0 +1,97 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "gmmdemux"
+
+description: GMM-Demux is a Gaussian-Mixture-Model-based software for processing sample barcoding data (cell hashing and MULTI-seq).
+keywords:
+  - demultiplexing
+  - hashing-based deconvoltion
+  - single-cell
+tools:
+  - "gmmdemux":
+      description: "GMM-Demux is a Gaussian-Mixture-Model-based software for processing sample barcoding data (cell hashing and MULTI-seq)."
+      homepage: "https://pypi.org/project/GMM-Demux/"
+      documentation: "https://github.com/CHPGenetics/GMM-Demux"
+      tool_dev_url: "https://github.com/CHPGenetics/GMM-demux"
+      doi: "10.1186/s13059-020-02084-2"
+      licence: ["MIT"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
```

```suggestion
        e.g. `[ id:'sample1' ]`
```
Single/paired info should not be relevant for this input type.